src/dropdown.option.ts

```diff
@@ -344,12 +351,33 @@ export default class GlideCoreDropdownOption extends LitElement {
                 ${pencilIcon}
               </button>`;
             })}
+            ${when(this.count, () => {
+              return html`<div
+                class=${classMap({
+                  'count-container': true,
+                  disabled: this.disabled,
+                  [this.privateSize]: true,
+                })}
+                data-test="count-container"
+              >
+                ${when(
+                  this.count! >= 1000,
+                  () => {
+                    return '999+';
```

Comment:
Maybe a silly question, I'm not sure, but should this be localized?

Reply:
Great question. I wonder how universal it is. At the very least, it should probably read right to left depending on the locale. I'll ask around.

Reply:
I talked to someone internally who specializes in localization. He said we should be good with the "+".